api/types/events/api.go

```diff
@@ -74,7 +74,7 @@ type AuditEvent interface {
 // Emitter emits audit events.
 type Emitter interface {
 	// EmitAuditEvent emits a single audit event.
```

Comment:
It this a good place to document that Emitters don't take a context because we always want the events to go through?

Should callers expect emitters to be sync or async?